modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -14,7 +14,8 @@ In the table, features are marked with the following statuses:
 |===
 |*OpenShift GitOps* 6+|*Component Versions*|*OpenShift Versions*|*Support status*
 
-|*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex*||
+|*Version*|*kam*|*Helm*|*Kustomize*|*Argo CD*|*ApplicationSet*|*Dex* |       |
+|1.4.0    |TBD  |3.7.1 |TBD        |2.2.2    |0.3.0           |TBD   |TBD    |GA
```

Comment:
@wtam2018: Would you please assign someone to provide the current version numbers for the items marked "TBD", here. Thank you.

Reply:
kam = 0.0.41
helm = 3.7.1
kustomize = 4.2.0
argo cd = 2.2.2
applicaitonset = 0.2.0
dex = 2.30.0
Reference: https://spaces.redhat.com/display/GITOPS/GitOps+Component+Matrix